es/13_0_Disenando_Bitcoin_Scripts_Reales.md

```diff
@@ -0,0 +1,21 @@
+# Capítulo 13: Diseño de scripts de Bitcoin reales
+
+Nuestros scripts de Bitcoin hasta la fecha han sido en gran parte ejemplos teóricos, porque todavía hemos estado juntando las piezas del rompecabezas. Ahora, con el repertorio completo de Bitcoin Script en la mano, estamos listos para profundizar en varios Scripts de Bitcoin del mundo real y ver cómo funcionan.
+
+## Objetivos de este capítulo
+
+Después de trabajar en este capítulo, un desarrollador podrá:
+
+ * Evaluar los scripts de Bitcoin del mundo real
+ * Crear scripts de Bitcoin del mundo real
+
+Los objetivos de apoyo incluyen la capacidad de:
+
+ * Comprender los scripts de Bitcoin existentes
+ * Comprender la importancia de las firmas
+
+## Tabla de contenido
+
+* [Sección uno: Escritura de scripts de rompecabezas](13_1_Escribiendo_Puzzle_Scripts.md)
```

Comment:
- `[Sección uno: Escribiendo scripts rompecabezas]`
- Change to include in #377.